common/roles.go

```diff
@@ -119,4 +119,6 @@ const (
 	SecretReaderRoleName = "SecretReader"
 	// Health Reader role - has permissions to read agent health information
 	HealthReaderRoleName = "HealthReader"
+	// Certificate Signer Role - has permissions to run sign and renew operations on Certificates
+	CertSignerRoleName = "CertificateSigner"
```

Comment:
Where are we defining the role permission?

Reply:
I'll add them to the defaultroles.go file of cloud agent repo. But that has a dependency on this file